api/api.py

```diff
@@ -46,10 +48,9 @@ def get_book_details():
             422,
         )
     # we get the ibsn, we check and we return a message according to the result
-    ibsn = request.args.get("ibsn")
-    if ibsn == "":
-        print("ibsn")
-        raise BadRequestException()
+    ibsn = request.args.get("ibsn", default=None, type=None)
```

Comment:
You do not need to specify default=None, type=None because this is the default behavior.